### PR TITLE
update zeit now link on event triggers page

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Minor tweaks for docs
### Description

- Fixed outdated url

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs update
- [ ] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [x] This change requires a change in the documentation. 
- [x] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

File: docs/graphql/manual/api-reference/schema-metadata-api/table-view.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Add or remove a table/view to Hasura GraphQL Engine's metadata using following A
 
 .. Note::
 
-   Only tables/views added to metadata are available for ``querying/mutating/subscribing`` data over **GraphQL** API.
+	Only tables/views added to metadata are available for ``querying/mutating/subscribing`` data over **GraphQL** API.
```
Comment (dsandip) [inline docs/graphql/manual/api-reference/schema-metadata-api/table-view.rst]: What does this change achieve?

Comment (dsandip) [reply docs/graphql/manual/api-reference/schema-metadata-api/table-view.rst]: @jamesgeorge007 Oh okay. That's cool. The change in `event-triggers.md` makes perfect sense :)

Comment (jamesgeorge007) [reply docs/graphql/manual/api-reference/schema-metadata-api/table-view.rst]: @dsandip Hope everything is fine now :+1:

**Comment by jamesgeorge007** [comment]
@dpisanu What's the status?

**Comment by dpisanu** [comment]
Uhmm, @jamesgeorge007 ... Status of what?

**Comment by jamesgeorge007** [comment]
@dpisanu I haven't heard from the maintainers for a while and was curious to know whether I should an further action on this PR!

**Comment by dpisanu** [comment]
@jamesgeorge007 ... Ah, but I am not a maintainer here. You had a typo in your tag and woke me from my slumber. 😋

**Comment by dsandip** [comment]
@jamesgeorge007 we'll merge this PR shortly.

File: event-triggers.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Serverless/cloud-function platforms covered by boilerplates:
 * [AWS Lambda](community/boilerplates/serverless-triggers/aws-lambda)
 * [Google Cloud Functions](community/boilerplates/serverless-triggers/google-cloud-functions)
 * [Azure Functions](community/boilerplates/serverless-triggers/azure-functions)
-* [Zeit Serverless Docker](community/boilerplates/serverless-triggers/zeit-serverless-docker)
+* [Zeit Serverless Docker](community/boilerplates/serverless-triggers/zeit-now)
```
Comment (shahidhk) [inline event-triggers.md]:
```suggestion
* [Zeit Now](community/boilerplates/serverless-triggers/zeit-now)
```

**Comment by shahidhk** [comment]
@jamesgeorge007 Thanks for the PR, please check the change request?

**Comment by jamesgeorge007** [comment]
@shahidhk Hope it is fine now :+1:

**Pull request merged**